### DISCUSSION TITLE
file back

### DISCUSSION THREAD

**Discussion by BinOmair** [community/community#140572] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Transmission team, I hope and I wish you put style files and group torrent files in folders like before, sometimes it's important to have it from to put colors on torrent it's easier to stop folder group than to select torrent color day by day, I hope it can be back, Thanks always...All the best of the best

**Comment by LiteBrite82** [comment]
Hi @BinOmair ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
